app/models/agent.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime
+from typing import Any, Dict, List, Optional
+
+from sqlalchemy import Column, String, func
+from sqlalchemy.dialects.postgresql import ARRAY, JSONB
+from sqlmodel import Field, Session, SQLModel, select
+
+from utils.slack import send_slack_message
+
+
+class Agent(SQLModel, table=True):
+    """Agent model."""
+
+    __tablename__ = "agents"
+
+    id: str = Field(primary_key=True)
+    # AI part
+    name: Optional[str] = Field(default=None)
+    model: str = Field(default="gpt-4o-mini")
+    prompt: Optional[str]
+    # autonomous mode
+    autonomous_enabled: bool = Field(default=False)
+    autonomous_minutes: Optional[int]
+    autonomous_prompt: Optional[str]
+    # if cdp_enabled, will load cdp skills
+    # if the cdp_skills is empty, will load all
+    cdp_enabled: bool = Field(default=False)
+    cdp_skills: Optional[List[str]] = Field(sa_column=Column(JSONB, nullable=True))
+    cdp_wallet_data: Optional[str]
+    cdp_network_id: Optional[str]
+    # if twitter_enabled, twitter_config will be checked
+    twitter_enabled: bool = Field(default=False)
+    twitter_config: Optional[dict] = Field(sa_column=Column(JSONB, nullable=True))
+    twitter_skills: Optional[List[str]] = Field(sa_column=Column(ARRAY(String)))
```

Comment:
the skill "categories" should be more generic, for example, should be social_skills -> twitter_skills

Reply:
I haven't figured out how to organize the configuration for general skills and skill sets yet. In any case, I want to treat Twitter and Telegram as "first-class" configurations; they are special and at the root level of the configuration. For example, Slack is configured within the skill sets and is not visible in the configuration definitions.